fix "comma at end of enumerator list" error with older compilers defaulting to C89

--- a/include/FLAC/format.h
+++ b/include/FLAC/format.h
@@ -512,7 +512,7 @@
 	FLAC__METADATA_TYPE_UNDEFINED = 7,
 	/**< marker to denote beginning of undefined type range; this number will increase as new metadata types are added */
 
-	FLAC__MAX_METADATA_TYPE = FLAC__MAX_METADATA_TYPE_CODE,
+	FLAC__MAX_METADATA_TYPE = FLAC__MAX_METADATA_TYPE_CODE
 	/**< No type will ever be greater than this. There is not enough room in the protocol block. */
 } FLAC__MetadataType;
 
